codeclimate-help/bin/run

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+underline() { printf "\e[4m%s\e[0m" "$*"; }
+
+cat <<EOM
```

Comment:
This take is probably fine for now, but right now I think one of the important missing features in the CLI is support for `subcommand --help` or `help subcommand`. I think supporting both of those is important.

Actually I guess this script could implement the latter one by just...running `subcommand --help`, so I guess it's really just the former case we should be addressing.